subprojects/core/src/test/groovy/org/gradle/initialization/DefaultCommandLineConverterTest.java

```diff
@@ -188,7 +188,13 @@ public void withShowStacktraceAndShowFullStacktraceShouldThrowCommandLineArgumen
     }
 
     @Test
-    public void withDryRunFlagSet() {
+    public void withDryRun() {
+        expectedDryRun = true;
+        checkConversion("-dry-run");
```

Comment:
:x: Should this be `--dry-run`? Is this test passing?

Reply:
Yeah, should be though the command line parser doesn't seem to care. Will change it.

Reply:
done